.github/workflows/manual-dev-deploy.yaml

```diff
@@ -30,3 +30,12 @@
             cf_username: ${{secrets.CF_USER}}
             cf_password: ${{secrets.CF_PASSWORD}}
             command: push funding-service-design-application-store-dev
+        - name: Apply network policy for notification service
```

Comment:
do we still need this?

Reply:
Throws an error if removed so probably yes